masterlist.yaml

```
@@ -7779,3 +7779,19 @@ plugins:
       - link: 'https://www.nexusmods.com/fallout4/mods/68889/'
         name: 'Extra Icons for FIS'
     after: [ 'FIS-Naming-Weap-Armo-EN.esp' ]
+
+  - name: 'Chameleon No Sound.esp'
+    url:
+      - link: 'https://www.nexusmods.com/fallout4/mods/1886'
+    msg:
+      - <<: *alreadyInOrFixedByX
+        subs: [ 'Alternative Chameleon Visuals' ]
+        condition: 'active("AltChameleonVisuals.esp") or active("NoChameleonFlash.esp")'
+        
+  - name: 'NoChameleonFlash.esp'
+    url:
+      - link: 'https://www.nexusmods.com/fallout4/mods/1886'
+    msg:
+      - <<: *alreadyInX
+        subs: [ 'Alternative Chameleon Visuals' ]
+        condition: 'active("AltChameleonVisuals.esp")'
```

`useOnlyOneX` would be more appropriate here since users may have a reason to use this plugin:

```suggestion
      - <<: *useOnlyOneX
        subs: [ 'Alternative Chameleon Visuals ESP' ]
        condition: 'many("(Alt|No)Chameleon(Flash|Visuals)\.esp")'
```